ImperatorToCK3/Source/Mappers/TagTitleMapper/TagTitleMapping.cpp

```diff
@@ -0,0 +1,30 @@
+#include "TagTitleMapping.h"
+#include "ParserHelpers.h"
+
+mappers::TagTitleMapping::TagTitleMapping(std::istream& theStream)
+{
+	registerKeys();
+	parseStream(theStream);
+	clearRegisteredKeywords();
+}
+
+std::optional<std::string> mappers::TagTitleMapping::tagRankMatch(const std::string& imptag, const std::string& rank) const
+{
+	if (impTag == imptag && ranks.contains(rank))
```

Comment:
I'd kind of prefer to see the exceptional case covered by the if statement, with the happy path separated by a blank line.

That is, please invert the logic.